Documentation/contributing/testing/ci.rst

```diff
@@ -137,8 +137,9 @@ This box will need to be updated when a new developer needs a new dependency
 that is not installed in the current version of the box, or if a dependency that
 is cached within the box becomes stale.
 
-Make sure that you update vagrant box versions in `test Vagrantfile <https://github.com/cilium/cilium/blob/master/test/Vagrantfile>`__
-and `root Vagrantfile <https://github.com/cilium/cilium/blob/master/Vagrantfile>`__ after new box is built and tested.
+Make sure that you update vagrant box versions in `vagrant_box_defaults.rb
+<https://github.com/cilium/cilium/blob/master/vagrant_box_defaults.rb>`__ after
+new box is built and tested.
 
 Once you change the image versions locally, create a branch named
 ``pr/update-packer-ci-build`` and open a PR ``github.com/cilium/cilium``.
```

Comment:
The PR itself is fine, but to ensure correct caching, the branch for such commits must be named `pr/update-packer-ci-build` on `github.com/cilium/cilium`, _not_ on a fork. We probably should update the docs to make that more clear, now that most users use their own forks for development.